src/ssh_cert_service/views/api_v1/api-spec/paths/key/verify.yml

```diff
@@ -0,0 +1,22 @@
+post:
+  summary: Verify key signatures
+  description: Verify that public key was used to sing the cert key.
```

Comment:
should red: 'Verify public and signature'

This checks that a given public and the certificate ... belong together, that it's signed by this service's CA and that it's valid (time constraint)

The CA private key is used to sign a pub key to generate a certificate .... the pubkey is the bit that get's signed by the Ca (the pubkey is not involved in any other way during the signing process)